### PR TITLE
Updated the environment variables

### PR DESCRIPTION
**Pull request by SanskritiHarmukh**
- [x] Added the SMTP optional and custom mailer config environment variables.

### CODE REVIEW

File: documentation/self-host/community-edition/install-and-build.mdx
Comment (balub) [inline documentation/self-host/community-edition/install-and-build.mdx:31]: Lets not merge this part just yet, the PR is still open and its not merged.

File: documentation/self-host/community-edition/install-and-build.mdx
Comment (balub) [inline documentation/self-host/community-edition/install-and-build.mdx:64]: Remove whitespace here.

File: documentation/self-host/community-edition/install-and-build.mdx
Comment (balub) [inline documentation/self-host/community-edition/install-and-build.mdx:104]: Lets put this here after the PR is merged.

**Review by balub — CHANGES_REQUESTED** [review]

File: documentation/self-host/community-edition/install-and-build.mdx
Comment (SanskritiHarmukh) [reply documentation/self-host/community-edition/install-and-build.mdx:31]: Cool

File: documentation/self-host/community-edition/install-and-build.mdx
Comment (SanskritiHarmukh) [reply documentation/self-host/community-edition/install-and-build.mdx:64]: Okay

File: documentation/self-host/community-edition/install-and-build.mdx
Comment (SanskritiHarmukh) [reply documentation/self-host/community-edition/install-and-build.mdx:104]: Got it.

**Review by balub — APPROVED** [review]

**Pull request merged**
